gym-game2048/gym_game2048/envs/game_2048.py

```diff
@@ -255,27 +246,19 @@ def reset(self):
         self.__total_score = 0
         self.__invalid_count = 0
         self.__total_count = 0
+        self.__num_obstacles = 0
         self.__add_two_or_four()
         self.__add_two_or_four()
 
-
-def moveTranslate(move):
-    if move == 0:
-        print("Up")
-    if move == 1:
-        print("Down")
-    if move == 2:
-        print("Right")
-    if move == 3:
-        print("Left")
-
-
-game = Game2048(board_size=4)
-game.reset()
-for _ in range(20):
-    move = np.random.randint(0, 4)
-    moveTranslate(move)
-    game.make_move(move)
-    game.confirm_move()
-    print(game.get_board())
-
+    def moveTranslate(self,move):
```

Comment:
Updated this function to return a string that can be stored in the environment info dictionary and printed from the main file